packages/voter-dapp/src/DrizzleLogin.js

```diff
@@ -8,6 +8,11 @@ import DrizzleInit from "./DrizzleInit.js";
 function DrizzleLogin(props) {
   const [isButtonDisabled, setButtonDisabled] = useState(false);
   const [drizzle, setDrizzle] = useState(null);
+  const [hasMetamask, setHasMetamask] = useState(false);
+
+  const getMetamask = () => {
+    window.open("https://metamask.io/", "_blank");
```

Comment:
I've never used `window.open()` to open a link, only `<a>` tag's with `url` and `target` props set. Cool stuff